matrix-meetings-bot/src/service/MeetingService.ts

```diff
@@ -604,6 +604,14 @@ export class MeetingService {
           )
         );
       }
+      const filteredWidgetIds = eventWidgetIds.filter(
```

Comment:
This doesn't work in all scenarios:

1. Only the NeoBoard is active.
2. Jitsi is added.
3. **Expected:** The layout of NeoBoard&Jitsi is installed. **Actual:** The layout Jitsi is installed.

-> I think you need to use `widgetIds` here, which is the list of all installed widgets after the update. `eventWidgetIds` only includes the ones that were added/removed by the caller.

The current code would also install the layout for `jitsi` if the widget was removed due to how this API is designed.